src/sources/en/wuxiablog.js

```diff
@@ -164,40 +156,9 @@ const parseChapter = async (novelUrl, chapterUrl) => {
 };
 
 const searchNovels = async searchTerm => {
-  const url = `${baseUrl}?search=${searchTerm}`;
-
-  const result = await fetch(url);
-  const body = await result.text();
-
-  const loadedCheerio = cheerio.load(body);
-
-  let novels = [];
-
-  loadedCheerio('#table')
-    .find('tr')
-    .each(function () {
-      const novelName = loadedCheerio(this).find('a').text().trim();
-
-      if (novelName) {
-        const novelCover = loadedCheerio(this).find('img').attr('src');
-
-        let novelUrl = loadedCheerio(this)
-          .find('a')
-          .attr('href')
-          .replace(baseUrl + 'novel/', '');
-
-        const novel = {
-          sourceId,
-          novelName,
-          novelCover,
-          novelUrl,
-        };
-
-        novels.push(novel);
-      }
-    });
+  showToast('Search is not available in this source');
```

Comment:
Is it not available anymore or could be temporarily broken?

Reply:
most likely temporarily broken, but I cannot confirm
![image](https://github.com/LNReader/lnreader/assets/26593485/c6be0459-ec01-4674-9b1c-52dcb2687a6c)

Reply:
Then we could keep the existing Searching logic as it should start working when the Source website fixes it.

Reply:
reverted to preserve previous search logic